fpn/fpndecimal.go

```diff
@@ -0,0 +1,100 @@
+package fpn
+
+import (
+	"github.com/nikolaydubina/fpdecimal"
+	"github.com/nikolaydubina/fpdecimal/constraints"
+)
+
+// Decimal is fixed-point decimal with 3 fractions.
+// Actual value scaled by 1000x.
+// Values fit in ~9 quadrillion.
+// Fractions lower than that are discarded in operations.
+// Max: +9223372036854775.807
+// Min: -9223372036854775.808
+type Decimal struct{ v int64 }
+
+var Zero = Decimal{}
+
+var multipliers = []int64{1, 10, 100, 1000, 10000, 100000, 1000000, 10000000, 100000000, 1000000000, 10000000000}
+
+var fractionDigits int8 = 3
+
+func SetFractionDigits(fraction int8) bool {
+	if fraction < 0 || fraction > 10 {
+		return false
+	}
+	fractionDigits = fraction
+	return true
+}
+
+func getInt64fractionDigits() int64 {
+	return multipliers[fractionDigits]
+}
```

Comment:
one liner for small func where fits
```suggestion
func getInt64fractionDigits() int64 { return multipliers[fractionDigits] }
```

Reply:
There are setting of my IDE :)